config/settings/qa.yml

```diff
@@ -0,0 +1,7 @@
+argo_url: 'https://argo-qa.stanford.edu'
+etd_url: 'https://etd-qa.stanford.edu'
+hydrus_url: 'https://sdr-qa.stanford.edu'
+preassembly_bundle_directory: '/dor/staging/integration-tests/image-test'
```

Comment:
we use the same directory?  Is there a downside/do we have to?

Reply:
I moved this into config just in case the value will vary between staging and QA.

Reply:
This moved from test setup to config just in case this value will vary between environments. The value is unchanged.